frontend/web/index-test.php

```diff
@@ -1,14 +1,19 @@
 <?php
+
+// Composer
+require __DIR__ . '/../../vendor/autoload.php';
+
+// Helpers
+require_once(__DIR__ . '/../../common/helpers.php');
+
+// Environment
 require(__DIR__ . '/../../tests/bootstrap.php');
 
 // NOTE: Make sure this file is not accessible when deployed to production
 if (YII_ENV !== 'test') {
     die('You are not allowed to access this file.');
 }
 
-// Environment
-require(__DIR__ . '/../../common/env.php');
```

Comment:
Why would you ignore the `env.php` file? It is widely used inside the application (it provides the `env()` function application-wide).

In fact, the tests are failing exactly due to this reason. Could you re-add this file again?

Reply:
env() function actually provided by common/helpers.php file, not common/env.php file.
common/env.php also contains environmental definitions that already bootstraped in tests/bootstrap.php:
`defined('YII_DEBUG') or define('YII_DEBUG', true);
defined('YII_ENV') or define('YII_ENV', env('YII_ENV', 'test'));
defined('YII_APP_BASE_PATH') or define('YII_APP_BASE_PATH', dirname(__DIR__));`

If i re-add common/env.php file then environmental constants will be defined twice, (in common/env.php and in tests/bootstrap.php) which is a little bit confusing.